x-pack/plugins/screenshotting/server/formats/pdf/pdf_maker/pdfmaker.ts

```diff
@@ -83,6 +83,8 @@ export class PdfMaker {
     const groupCount = this.content.length;
 
     // inject a page break for every 2 groups on the page
+    // TODO: Remove this code since we are now using Chromium to drive this
```

Comment:
Is this planned to do in this PR? It seems reasonable to.

Reply:
I'd like to do that in a separate PR mainly because I've been trying to focussing changes on print layout reports where that code path also touches other reports and not changing it now does not affect those.

Does that sound OK to you?